src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable semi */
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable no-alert */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
[OPTIONAL]
- Avoid deactivating too many linters, this may lead to a weak code structure. Don't worry about it now, but keep it in mind for the next projects.

Reply:
@mrjuda , Thank you for the feedback

I have updated the readme file. Other changes are well noted for future action